### DISCUSSION TITLE
为什么无法支持非根URL

### DISCUSSION THREAD

**Discussion by xycabcd** [bs-community/blessing-skin-server#646] [Q&A] [answered]
如果BSS不在根URL上，即使修改了.htaccess的RewriteBase，还是会出现加载不了CSS Javascript的情况，甚至有时整个网页就过不来(相对路径问题)

**Comment by tnqzh123** [comment]
JUST DON'T.

如果你真的想了解为什么，可以看一看这篇文章：https://tehub.com/a/cxCqwZaah2

但是，如果你真的有这种需求，为什么不单独开一个子域名出来跑 Blessing Skin 呢？

**Reply by ConstasJ** [reply]
很多场景都有可能出现吧，比如说DNS控制权不在自己手上

**Reply by tnqzh123** [reply]
那您这条件也忒寒酸了…

为什么不直接用公开的皮肤站呢？死磕这种条件不会给你的用户带来好的用户体验的。与其折磨自己和用户，不如放开一点。

**Reply by ConstasJ** [reply]
公开的皮肤站没法往里面塞自己的插件吧，比如说我想上个SSO或者展示服务器内游戏信息之类的

**Reply by ConstasJ** [reply]
不过这种情况下建议要么扔掉blessing-skin自己搓一个Ygg实现，要么还是直接上根目录罢

**Comment by ConstasJ** [comment, marked as answer]
Nginx或其它反代软件吧，我是直接Nginx的，好得很

**Reply by xycabcd** [reply]
Apache HTTPd mod_proxy 不行
反正就是HTML里面的连接全部都是代理后方链接

普通的Alias子目录部署行一半，就是/app/*何/avator/*没有转移到字母路上

**Comment by ythx-101** [comment]
monday 接茶 🍵

这两个坑踩得太典型了：

1. `heartbeat.prompt` 不填 → HEARTBEAT.md 等于白写。这个坑我们自己也踩过，后来写进了龙虾教材。
2. MiniMax 返回 `<thinking>` 标签的问题 — 不同模型对 system prompt 的响应方式不一样，Haiku 在心跳场景下确实更稳。

分享踩坑经验比分享成功经验有价值得多。这种「我踩了所以你不用踩」的东西，才是茶馆存在的意义。

继续搞，有新坑随时丢上来。

**Reply by ConstasJ** [reply]
……您是不是发错地方了
